themes/default/content/blog/next-level-iac-pulumi-automation-api/index.md

```diff
@@ -0,0 +1,357 @@
+---
+title: "Next-level IaC: Pulumi Automation API and API Economy"
+date: 2024-04-26
+meta_desc: Learn how to use Pulumi Automation API to build your own Service API and expose it to your organization.
+meta_image: meta.png
+
+authors:
+
+- engin-diri
+
+tags:
+
+- next-level-iac
+- automation-api
+- api-economy
+- cloudformation
+- terraform
+- hcl
+
+---
+
+When I am talking with our customers and users, there is one common theme that comes up: how can we make infrastructure as code (IaC) more accessible to our organization? This is a great question, and it is something that we have been thinking about a lot at Pulumi.
+
+We believe that the key to making IaC more accessible is to treat it as a first-class citizen in your organization.
+
+This means that you should be able to interact with your infrastructure in the same way that you interact with any other service in your organization. There are different ways to achieve this goal, be it through providing a self-service portal, building custom tooling, or integrating IaC into your existing workflows.
+
+This is where the Pulumi Automation API comes in and the concept of [API Economy](https://konghq.com/blog/enterprise/api-economy).
+
+## What is API Economy?
+
+![API Economy](api-economy.png)
+
+(Image [source](http://localhost:1313/blog/next-level-iac-pulumi-automation-api/api-economy.png))
+
+As a matter of fact: — APIs enable innovation.
+
+> Over [40% of large organizations](https://www.forbes.com/sites/patricksalyer/2021/05/04/api-stack-the-billion-dollar-opportunities-redefining-infrastructure-services--platforms/?sh=2d6c47f043f9) have 250+ APIs. And 71% plan to use even more APIs this year.
+
+API Economy is a term that is used to describe the way that organizations are using APIs to create new business models and revenue streams. APIs are the building blocks of modern software development, and they allow developers to build on  top of existing services and create new services that are greater than the sum of their parts.
+
+Some examples of API Economy are:
+
+- [Stripe](https://stripe.com/docs/api) - Allows companies to accept payments online
+- [Google Maps API](https://developers.google.com/maps) - Allows companies to use their maps, witout having to build their own
+- [Uber Eats API](https://developer.uber.com/docs/eats) - Allows companies to integrate with Uber Eats and offer food delivery
+
+But why is API Economy important for IaC? Because it allows you to treat your infrastructure as a service. The owner of a particular piece of infrastructure (like DNS, a database, or a Kubernetes cluster) can expose that infrastructure to the rest of the organization through an API. This API can be used to provision, update, and delete the infrastructure, as well as to query its current state. This allows developers to interact with the infrastructure in a programmatic way,  just like they would interact with any other service in the organization.
```

Comment:
I read this twice and still dont understand why I want to API wrap the infra. Maybe we need to give a few hypothetical examples to make the point